lineapy/db/relational.py

```diff
@@ -232,6 +232,19 @@ class PositionalArgORM(Base):
     argument = relationship(BaseNodeORM, uselist=False)
 
 
+class LPositionalArgORM(Base):
```

Comment:
It makes sense that `LineaCallNodeORM` is a separate class as its schema deviates from that of existing `CallNodeORM`. Yet, it is not clear to me why we would need other L-prefixed ORM classes (e.g., `LPositionalArgORM`) as their schema is the same as that of their counterparts (e.g., `PositionalArgORM`).

Reply:
there is some magic relationship involved in these classes. the regular PositionalArgORM contains references to the call_node.id (id column of call_node table). Because of this, i could not easily reuse the older ORM object for linea_call_node.id. There might be a cleaner way to ensure both these columns can be mapped simultaneously but for the prototype went with a new class.